src/conic_form.jl

```diff
@@ -26,14 +26,14 @@ _set_type(::MOI.ConstraintIndex{F,S}) where {F,S} = S
 cons_offset(conic::MOI.Zeros) = conic.dimension
 cons_offset(conic::MOI.Nonnegatives) = conic.dimension
 cons_offset(conic::MOI.SecondOrderCone) = conic.dimension
-cons_offset(conic::MOI.PositiveSemidefiniteConeTriangle) = Int64((conic.side_dimension*(conic.side_dimension+1))/2)
+cons_offset(conic::MOI.PositiveSemidefiniteConeTriangle) = Int((conic.side_dimension*(conic.side_dimension+1))/2)
```

Comment:
This should be `floor(conic.side_dimension * (conic.side_dimension+1), 2)`

Reply:
I replaced the implementation with MOI.dimension. I didn't want to remove cons_offset altogether in the same PR, maybe do so in a follow-up